### ISSUE TITLE
Potential card priority enhancement

### ISSUE THREAD

**Issue by abranon** [aiczk/Astro#7]
#6 was a really good addition to the plugin, I'm wondering if we can expand on this and add an additional priority system for the in-between minute card buffs;

We currently have the singular 2 minute buff priority system which works well with the 3 card buffs but card overcap system follows the same priority system despite different classes having better minute bursts. Could we add a secondary priority system for the cards outside of divination? Let me know if I need to reword this for clarity.

**Comment by aiczk** [comment]
Thanks for the Issue.
For example, if there was a melee under the effect of The Balance, Astro is throwing cards against Range, but you want to prioritize it so that it is thrown to the melee?
If I'm understanding you correctly, I can implement that!

**Comment by Nitrodroki** [comment]
Just a 'Priority outside of divination' thing I believe.

I think the idea is that a DRK for example is an outstanding bursting target, it is one of the worst targets to throw at outside of burst, so priority in and out of diviniation would be ideal yes.

**Comment by aiczk** [comment]
Oh, I see!
I'll make sure we can set priorities during and outside of bursts! I'll try to get that implemented ASAP!

**Comment by abranon** [comment]
Sorry for the late response but exactly how Nitro clarified, thanks !

**Comment by aiczk** [comment]
It's taking a while to debug, please wait a bit for the release X3

**Comment by aiczk** [comment]
Implemented in ver1.10!
Sorry it took so long to debug!

**Comment by Nitrodroki** [comment]
You are a god ! This is amazing.

**Comment by Nitrodroki** [comment]
An additionnal request, sorry for bombarding them would be to have a checkbox of 'Consider cards as 'In burst' while divination CD is about to be available/is available' behavior.

The idea is to be able to use the 2 first cards before divination in the opener and reopener on burst targets eventho divination has not been used yet.

https://media.discordapp.net/attachments/583963017746579477/938881667253293126/ast_ew_opener_4.png?width=663&height=671

Basically a 'Treat cards as in burst while divination is close to being ready' option.
